packages/x-charts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts",
-  "version": "6.18.3",
+  "version": "6.18.4",
```

Comment:
Nitpick: We usually sync the released version with the current root release version. 😉